src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -972,6 +973,12 @@ private function addAssetMapperSection(ArrayNodeDefinition $rootNode, callable $
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('importmap_integrity_algorithms')
+                            ->info('Array of algorithms used to compute importmap resources integrity.')
+                            ->beforeNormalization()->castToArray()->end()
+                            ->prototype('scalar')->end()
+                            ->defaultValue(['sha384'])
```

Comment:
```suggestion
                            ->defaultNull()
```

This would be a BC break imho.

But it will also slow down the build time / memory usage.. so I do not think this is something that should be enabled per default (like cors)

Reply:
Makes sense; guess the recipe would be a good place to put this default then?